frontend/src/components/DateRangePicker.js

```diff
@@ -1,12 +1,12 @@
-import React, { useState } from 'react';
```

Comment:
I moved this file up to the components folder because a couple of pages were using it. Hopefully that makes sense to do.